tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -113,6 +113,12 @@ func newSession(credentialPath, credentialAccountID, region string) (*awssession
 	})
 }
 
+// CreateWindowsVMWithPrivate creates a WindowsVM with private IP
+func (a *AwsProvider) CreateWindowsVMWithPrivateIP() (windowsVM types.WindowsVM, err error) {
+	usePrivateSubnet := true
```

Comment:
Why create a variable for this instead of just passing true to `a.createWindowsVM()`?
The function doc for `a.createWindowsVM()` denotes what the parameter is for

Reply:
How I abhor single use variables :-) And no, it does not really help with making the code readable ;-)

Reply:
For the record, I disagree. I would have not created a single use var if it is a string variable which conveys the argument name that I want to pass to method. Since it is a bool, it will be hard for people to tell unless they go the definition of method. But I changed it

Reply:
If you feel strongly I fine if you leave it in. We are not in the kernel after all :-)